src/scales/index.ts

```diff
@@ -100,3 +103,23 @@ export function isTransformable(scale: any): scale is TransformableScale {
   return (scale instanceof QuantitativeScale ||
   scale instanceof Category);
 }
+
+/*
+ * Whether scale should be comparable intuitively.
+ */
+export function isNotComparable(scale1: any, scale2: any): boolean {
```

Comment:
Is the updated more what you have in mind? The boolean pieces are this way to keep the scales not know about each other.

The bug itself was introduced as part of a hidden feature change in this PR: https://github.com/palantir/plottable/commit/f8945f075aef249347df45922961e54da8c02c8a#diff-813b01dbeb729f9cc4dce897dff649bdR93

I can also just revert those lines if we don't actually want that feature.